packages/grafana-data/src/datetime/common.ts

```diff
@@ -1,17 +1,46 @@
 import { TimeZone, DefaultTimeZone } from '../types/time';
 
+/**
+ * Type describing the date and time helper function options. Used for all the
+ * helper functions available to parse and format date and time values.
+ *
+ * @public
+ */
 export interface DateTimeOptions {
+  /**
+   * Specify if you want to override the timeZone used when parsing or formatting
+   * a date and time value. If no timeZone is set the default timeZone for the current
+   * user will be used.
+   */
   timeZone?: TimeZone;
 }
 
+/**
+ * Type to describe the time zone resolver function that will be used to access
```

Comment:
Unclear. Are you saying this is the type (noun) that describes the thing or that I should type (verb) to describe the thing. Please clarify.

Reply:
ah, it is the first one. Will clearify.